### PR TITLE
fix(global): :sparkles: in CoreIcon restProps sent to NativeIcon

### PR DESCRIPTION
**Pull request by PritamIT2023**
in CoreIcon restProps sent to NativeIcon

Ref: #325

## Description

Clearly describe the changes you've made in this pull request. Explain the purpose and reasoning behind the changes. Reference any relevant issues or discussions using keywords like "Fixes #<issue_number>" or "Resolves #<issue_number>".

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

File: package/components/dataDisplay/CoreIcon.js [hunk 1]
```diff
@@ -49,14 +46,13 @@ export default function CoreIcon(props) {
       color={color}
```
Comment (techoneel) [inline package/components/dataDisplay/CoreIcon.js]: remove those from props destructuring if it's value is not changed

Comment (techoneel) [reply package/components/dataDisplay/CoreIcon.js]: color, fontSize and component

**Review by techoneel — CHANGES_REQUESTED** [review]
please do the changes

File: package/components/dataDisplay/CoreIcon.js (hunk 1 above)
Comment (PritamIT2023) [reply package/components/dataDisplay/CoreIcon.js]: Done

**Review by techoneel — APPROVED** [review]
goo to go

**Pull request merged**